appsTransfer/src/main/res/layout/numbers_input_layout.xml

```diff
@@ -0,0 +1,161 @@
+<?xml version="1.0" encoding="utf-8"?>
+<android.support.v7.widget.GridLayout xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
You could use TableLayout with rows and remove the dependency
Not sure why you are using it.